src/Native/jitinterface/CMakeLists.txt

```diff
@@ -9,11 +9,16 @@ set(NATIVE_SOURCES
 
 if(WIN32)
     set(NATIVE_SOURCES ${NATIVE_SOURCES}
-        CodeHeap.cpp
-        JITCodeManager.cpp
         ../Runtime/coreclr/GCInfoDecoder.cpp
```

Comment:
This is one logical piece with CodeHeap and JITCodeManager - the whole thing (what's under `if (WIN32)`) should be under same condition.